spec/javascripts/app/views/help_view_spec.js

```diff
@@ -134,4 +141,30 @@ describe("app.views.Help", function(){
       });
     });
   });
-});
+
+  describe("chat section", function(){
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        this.chat_enabled = true;
+        this.view = new app.views.Help();
+        this.view.render();
+      });
+
+      it('should display the chat', function(){
+        expect(this.view.$el.find('a[data-section=chat]').length).toBe(1);
+      });
+    });
+
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        this.chat_enabled = false;
+        this.view = new app.views.Help();
+        this.view.render();
```

Comment:
@svbergerem : Unless I've missed something, [this does not work](https://travis-ci.org/diaspora/diaspora/jobs/51999707#L570)